digits/dataset/images/classification/test_views.py

```diff
@@ -442,6 +459,9 @@ class TestCreatedWide(TestCreated):
 class TestCreatedTall(TestCreated):
     IMAGE_HEIGHT = 20
 
+class TestCreatedAugmentation(TestCreated):
+    IMAGE_CHANNELS = 1
+
```

Comment:
Hum yeah I don't know why I added those lines, they are testing nothing. The real tests are done in `TestAugmentedDataset`. I will test the different augmentations yes.